src/main/webapp/components/query-selector/query-selector.js

```diff
@@ -12,31 +16,59 @@ import {
 } from '../index-cards'
 import AdvancedSearchQueryBuilder from '../query-builder/query-builder'
 const { useDrawInterface } = require('../../react-hooks')
+import useAnchorEl from '../../react-hooks/use-anchor-el'
 
-const useOpenClose = props => {
-  const [anchorEl, setAnchorEl] = React.useState(null)
-
-  const handleOpen = event => {
-    setAnchorEl(event.currentTarget)
-  }
+const defaultSearchForms = {
+  basic: BasicSearchQueryBuilder,
+  advanced: AdvancedSearchQueryBuilder,
+}
 
-  const handleClose = () => {
-    setAnchorEl(null)
-    if (props.onClose) {
-      props.onClose()
-    }
-  }
+const defaultSearchFormLabels = {
```

Comment:
Have you considered combining defaultSearchForms and defaultSearchFormLabels so that all of the needed information is one place? i.e. something like
```
basic: {
    label: 'Basic Search'
    queryBuilder: BasicSearchQueryBuilder
}
```

Reply:
At first I thought it would be inconvenient to have nested properties, but that doesn't actually seem like the case.